packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -0,0 +1,171 @@
+import { Construct, IResource, Resource, Stack } from '@aws-cdk/core';
+import { IFileSystem } from './efs-file-system';
+import { CfnAccessPoint } from './efs.generated';
+
+/**
+ * Represents an EFS AccessPoint
+ */
+export interface IAccessPoint extends IResource {
+  /**
+   * The ID of the AccessPoint
+   *
+   * @attribute
+   */
+  readonly accessPointId: string;
+
+  /**
+   * The ARN of the AccessPoint
+   *
+   * @attribute
+   */
+  readonly accessPointArn: string;
+}
+
+/**
+ * Represents the CreateionInfo
+ */
+export interface CreationInfo {
+  /**
+   * Specifies the POSIX user ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   */
+  readonly ownerUid: string;
+
+  /**
+   * Specifies the POSIX group ID to apply to the RootDirectory. Accepts values from 0 to 2^32 (4294967295).
+   */
+  readonly ownerGid: string;
+
+  /**
+   * Specifies the POSIX permissions to apply to the RootDirectory, in the format of an octal number representing
+   * the file's mode bits.
+   */
+  readonly permissions: string;
+}
+
+/**
+ * Represents the PosixUser
+ */
+export interface PosixUser {
+  /**
+   * The POSIX user ID used for all file system operations using this access point.
+   */
+  readonly uid: string;
+
+  /**
+   * The POSIX group ID used for all file system operations using this access point.
+   */
+  readonly gid: string;
+
+  /**
+   * Secondary POSIX group IDs used for all file system operations using this access point.
+   *
+   * @default - None
+   */
+  readonly secondaryGids?: string[];
+}
+
+/**
+ * Properties for the AccessPoint
+ */
+export interface AccessPointProps {
+  /**
+   * The efs filesystem
+   */
+  readonly fileSystem: IFileSystem;
+
+  /**
+   * Specifies the POSIX IDs and permissions to apply to the access point's RootDirectory. If the
+   * `RootDirectory` > `Path` specified does not exist, EFS creates the root directory using the `CreationInfo`
+   *  settings when a client connects to an access point.
+   *
+   * @default - None
```

Comment:
What does the default 'None' mean? 

How does EFS behave if the `path` specified does not exist and no `creationInfo` is specified?

Reply:
If `path` not specified then the root(/) will be used instead and the root(/) always exists so we don't need `creationInfo`  in this case.

Reply:
I understand that part. My question is - what happens when path specified and creationInfo is not?

Reply:
According to the document

>If you do not provide CreationInfo and the specified RootDirectory > Path does not exist, attempts to mount the file system using the access point will fail. 

Let me add this note in the docstring and also add a check to throw an Error just in case. Thanks for the heads-up.

Reply:
```suggestion
   * @default - None. The directory specified by `path` must exist.
```